src/commands/run.js

```diff
@@ -28,4 +57,24 @@ RunCommand.description = 'Runs the act locally in the current directory.\n' +
     'the act input, as well as all other data in key-value stores, datasets or request queues to be stored in the "apify_local" directory, ' +
     'rather than on the Apify platform.';
 
+RunCommand.flags = {
+    'purge-stores': flagsHelper.boolean({
+        char: 'p',
+        description: 'It deletes all default local folder stores without "INPUT.*" before a run.',
+        required: false,
+    }),
+    'purge-queue': flagsHelper.boolean({
+        description: 'It deletes default page queue before a run.',
+        required: false,
+    }),
+    'purge-dataset': flagsHelper.boolean({
+        description: 'It deletes default datatset folder before a run.',
+        required: false,
+    }),
+    'purge-key-value-store': flagsHelper.boolean({
+        description: 'It deletes all files from default key-value store folder without "INPUT.*" file before a run.',
```

Comment:
Deletes all records from the default key-value store in the local directory before the run starts, except for the "INPUT" key.